cmd/bridge/main.go

```diff
@@ -204,6 +206,10 @@ func main() {
 		}
 	}
 
+	if *fInactivityTimeout < 300 {
+		log.Warning("Flag inactivity-timeout is set to less then 300 seconds and will be ignored!")
```

Comment:
We should fail if `fInactivityTimeout` is set and `fK8sAuth` is *not* `oidc` or `openshift`.

Reply:
@spadgett comment addressed. PTAL